src/UI/Chat.module.css

```diff
@@ -56,7 +48,7 @@
 	height: 100%;
 	border: none;
 	background-color: transparent;
-	color: var(--parley-button-label-text-color, #FFF);
```

Comment:
Dit was de verkeerde naam of is hier nu echt de var veranderd waarmee hij gezet wordt

Reply:
Deze bestond ook niet en `--parley-button-label-color` wel